All_years/03-Costs/Costs-04-home_care.R

```diff
@@ -0,0 +1,138 @@
+#####################################################
+# Costs - Home Care
+# Author: James McMahon
+# Date: May 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input - hc_costs.xlsx
+# Description - Update the costs in hc_costs.xlsx first
+# These came from Improvent Service
+# Table: Cash Num-Den_Indi
+# Columns: XQ-XS
+# https://www.improvementservice.org.uk/benchmarking/explore-the-data
+#####################################################
+
+# Packages #
+
+library(dplyr)
+library(ggplot2)
+library(tidyr)
+library(purrr)
+
+costs_dir <- fs::path("/conf/hscdiip/SLF_Extracts/Costs/")
+hc_costs_path <- fs::path(costs_dir, "costs_hc_lookup.rds")
+hc_costs_path_old <- fs::path(costs_dir, "costs_hc_lookup_pre-Jun_2022.rds")
```

Comment:
I wonder if it is worth specifying latest_update or previous_update as an object here? then when it comes to running this we know to update the 'old' path? Although i realise we will want to convert this into the r conversion project anyway so this might not be necessary

Reply:
Yep, good point the June_2022 bit should be in a `latest_update` variable.